<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflSettingsWidget</class>
 <widget class="QWidget" name="ReflSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings Tab</string>
  </property>
  <layout class="QVBoxLayout" name="settingsMainLayout" stretch="1,1">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="expSettingsGroup">
     <property name="title">
      <string>Experiment Settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,2">
      <item>
       <layout class="QGridLayout" name="expSettingsLayout0" rowstretch="1,1,1,1,1,1,1,1" columnstretch="1,1,1,1">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="analysisModeLabel">
          <property name="text">
           <string>AnalysisMode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="transmissionRunsLabel">
          <property name="text">
           <string>Transmission run(s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="transmissionRunsEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="startOverlapLabel">
          <property name="text">
           <string>StartOverlap</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="startOverlapEdit"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="endOverlapLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>EndOverlap</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="endOverlapEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="polCorrLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>PolarisationCorrections</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="polCorrComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PNR</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="CRhoLabel">
          <property name="text">
           <string>CRho</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="CRhoEdit"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="CAlphaLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>CAlpha</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="CAlphaEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="CApLabel">
          <property name="text">
           <string>CAp</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="CApEdit"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="CPpLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>CPp</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="CPpEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="momentumTransferStepLabel">
          <property name="text">
           <string>dQ/Q</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="momentumTransferStepEdit"/>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="scaleFactorLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="scaleFactorEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="stitchLabel">
          <property name="text">
           <string>Stitch1DMany</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="summationTypeLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>SummationType</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="reductionTypeLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>ReductionType</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="analysisModeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>PointDetectorAnalysis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MultiDetectorAnalysis</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="summationTypeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>SumInLambda</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SumInQ</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="reductionTypeComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DivergentBeam</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NonFlatSample</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="getExpDefaultsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Get Defaults</string>
        </property>
        <property name="fixedWidth" stdset="0">
         <number>70</number>
        </property>
        <property name="fixedHeight" stdset="0">
         <number>100</number>
        </property>
        <layout class="QHBoxLayout" name="getExpDefaultsButtonLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="instSettingsGroup">
     <property name="title">
      <string>Instrument Settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2">
      <item>
       <layout class="QGridLayout" name="instSettingsLayout0" rowstretch="1,1,1,1,1,1" columnstretch="1,1,1,1">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="5" column="0">
         <widget class="QLabel" name="correctDetectorsLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>CorrectDetectors</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QComboBox" name="detectorCorrectionTypeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>VerticalShift</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RotateAroundSample</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lamMinLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>LambdaMin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="monIntMinLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MonitorIntegralMin</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="I0MonIndexEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>154</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lamMinEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>154</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="procInstEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="I0MonIndexLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>I0MonitorIndex</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="monBgMaxEdit"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="procInstLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>ProcessingInstructions</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="lamMaxLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>LambdaMax</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="monBgMaxLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MonitorBackgroundMax</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="lamMaxEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="intMonCheckBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="monBgMinLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MonitorBackgroundMin</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="monBgMinEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="monIntMaxLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>MonitorIntegralMax</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="monIntMinEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>154</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="intMonCheckLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>IntegratedMonitors</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="monIntMaxEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="correctDetectorsCheckBox">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="detectorCorrectionTypeLabel">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>DetectorCorrectionType</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="getInstDefaultsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Get Defaults</string>
        </property>
        <property name="fixedWidth" stdset="0">
         <number>70</number>
        </property>
        <property name="fixedHeight" stdset="0">
         <number>100</number>
        </property>
        <layout class="QHBoxLayout" name="getInstDefaultsButtonLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
